Orange/widgets/utils/itemmodels.py

```diff
@@ -222,39 +239,34 @@ def __setitem__(self, i, value):
     def wrap(self, table):
         self.beginResetModel()
         self._table = table
+        rows, cols = range(len(table)), range(len(table[0]) if table else 0)
+        self._roleData = [[{} for _ in cols] for _ in rows]
```

Comment:
I was following PyListModel example (except where noticeably cleaner). If 100,000,000 rows are added to PyListModel, its `_other_data` grows to overwhelming 29 GB.